Targets/WordPress/Sources/AppDelegate+WordPress.swift

```diff
@@ -0,0 +1,37 @@
+import UIKit
+
+class AppDelegate: WordPressAppDelegate {
+    override func makeBuildSettings() -> BuildSettings {
+        let configuration = XcodeBuildConfiguration.current
+        return BuildSettings(
+            itunesAppID: "335703880",
+            pushNotificationAppID: configuration.pushNotificationAppID
+        )
+    }
+}
+
+private enum XcodeBuildConfiguration {
+    case debug
+    case `internal`
+    case alpha
+    case release
+
+#if DEBUG
+    static let current: XcodeBuildConfiguration = .debug
+#elseif INTERNAL_BUILD
+    static let current: XcodeBuildConfiguration = .internal
+#elseif ALPHA_BUILD
+    static let current: XcodeBuildConfiguration = .current
+#else
+    static let current: XcodeBuildConfiguration = .release
+#endif
+
+    var pushNotificationAppID: String {
+        switch self {
+        case .debug: "org.wordpress.appstore.dev"
+        case .internal: "org.wordpress.internal"
+        case .alpha: "org.wordpress.alpha"
+        case .release: "org.wordpress.appstore"
+        }
+    }
+}
```

Comment:
I _might_ be possible to remove some duplication here, e.g. by pushing `XcodeBuildConfiguration` to WordPressGuts and then having a `PushNotificationAppIDProviding` protocol with a method that takes a `XcodeBuildConfiguration` input and returns the id `String`. 

But.. it feels a bit like yack shaving and it also forces each app target to have the same build configuration, which we might not want.

(_A good reminder that not all duplication should be removed._)

Reply:
Also, `WordPressGuts` shouldn't have any conditional compilation. 

> have the same build configuration

Yeah, this is part of the reason I wanted to decouple it from the framework. The app are free to determine how they want to define the settings.

The original compiler flags (`INTERNAL_BUILD`) are also duplicated and are defined on a per-app basis, so there is that.